app/models/vote.rb

```diff
@@ -16,6 +16,7 @@ class Vote < ApplicationRecord
     "O" => "Off-topic",
     "A" => "Already Posted",
     "S" => "Spam",
+    "U" => "Broken Link",
```

Comment:
```suggestion
    "B" => "Broken Link",
```

This'll help me instantly recognize it in ad-hoc queries.